lib/java/spring/auth/src/main/java/co/airy/spring/auth/AuthConfig.java

```diff
@@ -49,8 +49,8 @@ public AuthConfig(@Value("${systemToken:#{null}}") String systemToken,
                       @Value("${jwtSecret:#{null}}") String jwtSecret,
                       List<IgnoreAuthPattern> ignorePatternBeans,
                       ConfigProvider configProvider,
-                      @Value("${oidc.logoutSuccessUrl:/ui/}") String logoutSuccessUrl,
-                      @Value("${oidc.loginSuccessUrl:/ui/}") String loginSuccessUrl
+                      @Value("${oidc.logoutSuccessUrl:/control-center/}") String logoutSuccessUrl,
```

Comment:
This one should be changed to `/start`

Reply:
we are discussing a bit because it's not really clear where the start should go and how it should behave

Reply:
we will change it to just `/` since it will redirect to the correct start page @chrismatix